src/components/config/index.ts

```diff
@@ -108,6 +109,18 @@ export const setupConfigFromFile = async (flags: any, watch: boolean) => {
       updateConfig(parsed)
     })
   }
+
+  if (flags['create-config'] && (flags.har || flags.postman)) {
+    const file = flags.output || 'monika.json'
```

Comment:
Good idea to have default filename like `monika.json`. Cosmetic comment, feel free to ignore/resolve immediately: can default filename have better name to show that particular file is configuration for Monika?

Reply:
this is due to the current system will search the name `config.json` or `monika.json` as default configuration file and changing this as default would need to change the configuration in config generator also.